src/nss-resolve/nss-resolve.c

```diff
@@ -30,14 +30,15 @@
 #include "in-addr-util.h"
 #include "macro.h"
 #include "nss-util.h"
+#include "resolved-def.h"
 #include "string-util.h"
 #include "util.h"
 #include "signal-util.h"
 
 NSS_GETHOSTBYNAME_PROTOTYPES(resolve);
 NSS_GETHOSTBYADDR_PROTOTYPES(resolve);
 
-#define DNS_CALL_TIMEOUT_USEC (45*USEC_PER_SEC)
+#define DNS_CALL_TIMEOUT_USEC SD_RESOLVED_QUERY_TIMEOUT_USEC
```

Comment:
please drop DNS_CALL_TIMEOUT_USEC and use the new macro wherever it was used before